man/systemd-analyze.xml

```diff
@@ -253,6 +259,10 @@ NAutoVTs=8
     <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
     The hardware watchdog is not affected by this setting.</para>
 
+    <para><command>systemd-analyze timespan</command> parses a time span and outputs the equivalent value in microseconds.
+    The time span should adhere to the same syntax documented in <citerefentry><refentrytitle>systemd.time</refentrytitle><manvolnum>7</manvolnum></citerefentry>.
+    Values without associated magnitudes are parsed as microseconds.</para>
```

Comment:
almost all our time settings default to second magnitude. I think this should too, to be least surprising.